node/node.go

```diff
@@ -43,10 +42,10 @@ type Node struct {
 	Type          node.Type
 	Network       params.Network
 	Bootstrappers params.Bootstrappers
-	Config        *node.Config
+	Config        *Config
 
-	// CoreClient provides access to a Core node process.
-	CoreClient core.Client `optional:"true"`
+	// // CoreClient provides access to a Core node process.
+	// CoreClient core.Client `optional:"true"`
```

Comment:
wait what?

Reply:
That's an artifact from my latest commit. We _have to_ remove it

Reply:
which commit? Can you please link?